sites/intro-to-rails/intro-to-rails.step

```diff
@@ -79,6 +79,22 @@ The editor will automatically set up like this once it's ready.
 
 Congratulations! You're ready to start coding.
 
+Resuming
```

Comment:
Should this be treated as some sort of header?

Reply:
couldnt find any existing examples of sub-headers in the curriculum (hierarchically, it'd need to be an h4) but
 `#### Resuming` seems to do the right thing, wdyt?:

![image](https://github.com/railsbridge-boston/docs/assets/222655/2a8a9740-c9da-4c0d-8f42-ffa54945447c)